Source/Facebook/FacebookApiException.cs

```diff
@@ -80,10 +80,12 @@ public FacebookApiException(string message, string errorType, int errorCode)
         /// <param name="errorType">Type of the error.</param>
         /// <param name="errorCode">Code of the error.</param>
         /// <param name="errorSubcode">Subcode of the error.</param>
-        public FacebookApiException(string message, string errorType, int errorCode, int errorSubcode)
+        public FacebookApiException(string message, string errorType, int errorCode, int errorSubcode, string errorUserTitle, string errorUserMsg)

```

Comment:
This would result in a breaking change. Can we remove this from the constructor and only use it from the properties. Since there is a public setter we wouldn't need any changes in the constructor.